tests/infer/test_inference.py

```diff
@@ -304,18 +307,16 @@ def guide():
             beta_q_log = pyro.param("beta_q_log",
                                     Variable(self.log_beta_n.data - 0.143, requires_grad=True))
             alpha_q, beta_q = torch.exp(alpha_q_log), torch.exp(beta_q_log)
-            pyro.sample("p_latent", dist.beta, alpha_q, beta_q)
-            pyro.map_data("aaa", self.data, lambda i, x: None, batch_size=self.batch_size)
+            pyro.sample("p_latent", beta, alpha_q, beta_q)
 
         adam = optim.Adam({"lr": .001, "betas": (0.97, 0.999)})
         svi = SVI(model, guide, adam, loss="ELBO", trace_graph=False)
 
-        for k in range(10001):
+        for k in range(n_steps):
```

Comment:
@martinjankowiak The only change I made to step counts was to change this from 10001 to 10000. What was the extra 1 for?

Reply:
nothing important just checking. the 1 was so it'd print on the last step.... (so irrelevant)